Missing `npm install` in the release workflow

The release fails with the following error:
```bash
Error: Unable to publish to npm: npm publish --otp 720449 --tag latest returned code 127 and signal null
STDOUT: 
> fastify-auth0-verify@0.7.2 prepublishOnly
> npm run ci


> fastify-auth0-verify@0.7.2 ci
> npm run lint && jest --coverage --ci --coverageReporters=json && tsd


> fastify-auth0-verify@0.7.2 lint
> eslint index.js test.js


STDERR: sh: 1: eslint: not found
npm ERR! code 127
npm ERR! path /home/runner/work/fastify-auth0-verify/fastify-auth0-verify
npm ERR! command failed
npm ERR! command sh -c npm run ci

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2022-04-07T15_14_25_952Z-debug-0.log
```
The reason is that `npm install` is not run before running `npm run ci` step. To solve this issue a custom build step needs to be added to the release workflow.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -13,7 +13,7 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
       - run: npm install
-      - run: npm run ci
+      - run: npm run test:ci
       
   automerge:
     needs: build

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -16,7 +16,9 @@ jobs:
     steps:
       - uses: nearform/optic-release-automation-action@v2
         with:
-          github-token: ${{secrets.github_token}}
+          github-token: ${{ secrets.github_token }}
           npm-token: ${{ secrets[format('NPM_TOKEN_{0}', github.actor)] || secrets.NPM_TOKEN }}
           optic-token: ${{ secrets[format('OPTIC_TOKEN_{0}', github.actor)] || secrets.OPTIC_TOKEN }}
           semver: ${{ github.event.inputs.semver }}
+          build-command: |
+            npm install

--- a/package.json
+++ b/package.json
@@ -38,9 +38,9 @@
   "types": "index.d.ts",
   "scripts": {
     "test": "jest test --coverage --coverageReporters=html --coverageReporters=text && tsd",
-    "ci": "npm run lint && jest --coverage --ci --coverageReporters=json && tsd",
+    "test:ci": "npm run lint && jest --coverage --ci --coverageReporters=json && tsd",
     "lint": "eslint index.js test.js",
-    "prepublishOnly": "npm run ci",
+    "prepublishOnly": "npm run test:ci",
     "postpublish": "git push origin && git push origin -f --tags",
     "tsd": "tsd"
   },

